v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER shoudl be delcared as a config in the setup-credentials util
+
+source /etc/environment
+
+etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```

Comment:
Thanks @chr0n1x @matthewdfuller for the guidance.  I've applied the changes to the infrastructure repo as well via pull request https://github.com/adobe-platform/infrastructure/pull/31 per @matthewdfuller instruction.